schema.graphql

```diff
@@ -850,4 +856,19 @@ input StringSearchOptions {
 """
 A field whose value is a generic Universally Unique Identifier: https://en.wikipedia.org/wiki/Universally_unique_identifier.
 """
-scalar UUID
\ No newline at end of file
+scalar UUID
+
+type User {
+  """The address of the user"""
+  address: String!
+
+  """The avatar of the user"""
+  avatar: String!
```

Comment:
Avatar and display name can be undefined I assume?

Reply:
Resolved in https://github.com/hypercerts-org/hypercerts-api/pull/168/commits/bd5b11c3a23a262de2d68bd710baaeaff3f98a5f